### PR TITLE
Added tons of clones and reimplementations

### PR DESCRIPTION
**Pull request by wonea**
A few smaller cleanups.

### CODE REVIEW

**Comment by wonea** [comment]
Finished adding games from this batch. Not sure why it won't compile.

**Comment by wonea** [comment]
Fixes #132

**Comment by wonea** [comment]
Fixes #133

**Comment by piranha** [comment]
Wow, man, you're rocking it. Let me take a look at what fails...

File: games.yaml [hunk 1]
```diff
@@ -2041,6 +2211,35 @@
...
+- name: [Utopia, "Utopia_(video_game)"]
+  clones:
+    - name: Utopia (video game)
+      added: 2015-04-05
+      url: http://apps.gamejs.org/newutopia/
+      repo: https://github.com/gamejs/newutopia
+      info: stalled development, JavaScript
+
+- name: [Warlords II, "Warlords_(game_series)#Warlords_II"]
+  clones: FreeLords
```
Comment (piranha) [inline games.yaml]: You have to put "FreeLords" as a `name` property in the mapping, in other case YAML just can't parse the file.

File: games.yaml [hunk 2]
```diff
@@ -2041,6 +2211,35 @@
...
+      info: active development, Java, GPL2
+
+- name: [Uninvited, "Uninvited (video game)"]
+  clones:
+    - name: uninvited
+      added: 2015-04-04
+      repo: https://github.com/DavidGriffith/uninvited
+      info: sporadic development, Inform, Artistic License 2.0
+
+- name: [Utopia, "Utopia_(video_game)"]
```
Comment (piranha) [inline games.yaml]: We don't have underscores anywhere else, so I don't think they are needed here. Plus a name of clone in the next line is a bit weird. :)

File: games.yaml [hunk 3]
```diff
@@ -643,6 +697,15 @@
...
...1uwjALoXgLMTIALgs/1uEABj7Wv73zz/EQAIAATAvuW7F4DYBDTLvvb86QLAndMFQO2+2fYHgQDw6b9bPJwfndfut5b3EiC97f9NdAKOLt+9ACAAEAAdIADm2B8EAgABcLYAGP0IPrp89wKwdgLV1jfL/mr9t9bjvmhlEQCj7GvLDxIEpwsAr3aMFgStiXCWAGgt79WPenkEQOcCgABAAHSAAOizry2PAPhxv7Y8AuD9/bPKxwuAssBqCXwTAVDery3vncBnt3+18qPi2ow2cXklzlMFwOj6quPkbAEQ7RfRzuRH6LMFwHi/RCfg6PK9AxABsFX5UXFtBgHQZ9+rXc3jBAEw0i+iHQRAZ7+iE3B0+d4BuHkC3K39o/rf65dmehNVdAJexb63X9XzP1YASOXL66PsjvKLaL8zce4uACQ7CACpfO8APCQB7tJ+BEBneQQAAmCA3VF+Ee0jAJILAG09zon/94DXYS1Xu99qt1bv6gm813/e9biPHa9EJdXTmih7y9fu9/ZDa/u75/3nRKdNzNZy1vaMar+1Hnf/C4nOmpi9y9XuX6X9CAC3hQABgABoAAHg4wcEwMf6RrXfWo+7/xEAXe2/vAMCAAAAWxDeAAAAAJiPT0W9j4CiHiFBEQf7IyRVOal8enq3sCgfugUYTdgW1iH0+i+6fAdeDkQAnAACIIjoBJi9/OYgAGL9F12+g17H2RK29SUYqTz4EPUSTHqsCas8T/m+8pvT+xJudnr9F13egV4HIgBOAAEQRHQCzF5+cxZIIFsTncAXiF+vAxEAJzFKALAVUEGbwLTnKW8rvzkLJJCt0fpPe352eQd6HYgAOAkEwGSiE2D28puDAJjjPwSA6MjPiVyb2HkZMBZvAVATAtH9XAYpgdUSGeV9yh+ClEhI/D5+sybw0eUd8HIkAuAEEACTiU6A2csfAgJgjt8QAGrH9iVwBEAMvAQ4GW3iKu+nvE/5zbEmfISAr/+iyzswyrEIgB1BAEwmOgFmL785CySQrYlO4AvEb5Rj+xI3iT8GBMBkohNg9vKbs0AC2ZroBL5A/EY5FgGwIwiAyUQnwOzlN2eBBLI10Ql8gfh5OZJH/idgTejaciR+ASmRSYmL8n3lN0dKJCT+sf6LLt+BlwMRACeAAAgiOgFmL785CIBY/0WXb+X/AfoGzIAXoPKXAAAAAElFTkSuQmCC
```
Comment (piranha) [inline games.yaml]: Hmm... I'd rather have an image hosted on imgur, that a long `data:` URI in the yaml file. :)

**Comment by wonea** [comment]
Okay, got rid of the games.yaml errors.

Now got (from the build server);

 File "/home/travis/build/piranha/osgameclones/index.html", line 55, in template
{% set wikilink = "http://en.wikipedia.org/wiki/" + wiki %}
TypeError: cannot concatenate 'str' and 'list' object

**Comment by piranha** [comment]
@wonea that means somewhere you have three items in a list or something. I've added some validation to yaml-parsing code, so that it's easier to understand the problem, and will comment right now on both of the problems I found.

File: games.yaml [hunk 4]
```diff
@@ -1670,8 +1797,19 @@
...
+    - url: http://sourceforge.net/projects/stuntcarremake/
+      repo: git://git.code.sf.net/p/stuntcarremake/code
+      name: Stunt Car Racer Remake
+      added: 2015-04-04
+      info: stalled development, C++
+      media:
+        - url: https://a.fsdn.com/con/app/proj/stuntcarremake/screenshots/SCR2.png
 
-- name: [Stunts, Stunts (video game)]
+- name:
```
Comment (piranha) [inline games.yaml]: This should be `names:`.

File: games.yaml [hunk 5]
```diff
@@ -1670,8 +1797,19 @@
...
+      name: Stunt Car Racer Remake
+      added: 2015-04-04
+      info: stalled development, C++
+      media:
+        - url: https://a.fsdn.com/con/app/proj/stuntcarremake/screenshots/SCR2.png
 
-- name: [Stunts, Stunts (video game)]
+- name: 
+  - [Stunts, Stunts (video game)]
+  - [4D Sports Driving]
```
Comment (piranha) [inline games.yaml]: This shouldn't be wrapped in brackets, since it's only one string and template fails there (`wiki` becomes `None`).

Comment (wonea) [inline games.yaml]: That sorted it! Thanks for your patience.

**Comment by piranha** [comment]
Ha-ha, thank you for doing all that work!

**Pull request merged**